src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.cc

```diff
@@ -295,17 +297,18 @@ media_manager::FromMicToFileRecorderThread::FromMicToFileRecorderThread(
     : threads::ThreadDelegate()
     , impl_(new Impl("-o", "-t", output_file, duration)) {
   LOGGER_AUTO_TRACE(logger_);
-  sleep_thread_ = FromMicToFileRecorderThreadPtr(
-      "AudioFromMicSuspend",
-      new timer::TimerTaskImpl<FromMicToFileRecorderThread>(
+  sleep_thread_ =
```

Comment:
@dev-gh `FromMicToFileRecorderThreadPtr` was a smart pointer that removes timer finally, but now seems like you have leak